pvr.iptvsimple/changelog.txt

```diff
@@ -1,3 +1,6 @@
+v20.3.2
```

Comment:
As you are adding functionality the minor version should be increased. So it should be `20.4.0`.